proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -239,10 +282,20 @@ message MetricSample {
   //   to determine the SoC of the battery using its voltage, identifying the
   //   `dc_battery_0` connection is important.
   // - A sensor unit might report temperature from different connected probes
-  //   (e.g., `temp_sensor_outdoor`, `temp_sensor_indoor`).
+  //   (e.g., `temp_sensor_outdoor`, `temp_sensor_transformer`).
+  //
+  // !!! note "Default Connection Behavior"
+  //     If this field is unset, it is implicitly assumed that the component
+  //     only has a single connection for this metric, and the connection is
+  //     uniquely identified by the component itself.
   //
-  // In cases where the component has just one such connection for a metric,
-  // or the context is otherwise unambiguous, this field is not expected to
-  // be present, because the connection is implicit.
-  optional string connection = 5;
+  // !!! example
+  //     ```
+  //     // DC power from battery string 0 in a hybrid inverter
+  //     source {
+  //       category: BATTERY
```

Comment:
Is this missing the whole prefix `METRIC_CONNECTION_CATEGORY_` on purpose? If it is done like this usually to keep examples short, LGTM.